automatic/git.install/tools/helpers.ps1

```diff
@@ -1,27 +1,29 @@
-﻿function Get-InstallKey()
+﻿function Get-InstallOptions( [HashTable]$pp )
 {
-    $keyName = 'Git_is1'
-    $installKey = if ((Get-OSArchitectureWidth 64) -and $env:chocolateyForceX86 -eq 'true') {
-             "HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\$keyName"
-    } else { "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$keyName" }
+    $options = @()
```

Comment:
Since you are using custom arguments, can you please add a comment here with a link to any page that mentions the available options that you are using?

Reply:
Done, I hope it is good enough but there is no official documentation.

Reply:
Thank you for getting those added. I wish there was an official page, but it is what it is.

I think that was the reason why we maybe used the registry initially, since we could see those. But hadn't seen the source for the installer.